pkg/rules/otsdk/trace-context/ot_trace_context.go

```diff
@@ -137,11 +137,11 @@ func traceContextAddSpan(span trace.Span) {
 
 func GetTraceAndSpanId() (string, string) {
 	tc := GetTraceContextFromGLS()
-	if tc != nil {
-		ctx := tc.(*traceContext).tail().SpanContext()
-		return ctx.TraceID().String(), ctx.SpanID().String()
+	if tc == nil || tc.(*traceContext).tail() == nil {
+		return "", ""
 	}
-	return "", ""
+	ctx := tc.(*traceContext).tail().SpanContext()
```

Comment:
Later we may make the files in `rules` directory a go module in order to add unit tests for them. We can only add some integration tests now. @y1yang0